django_select2/util.py

```diff
@@ -101,7 +106,7 @@ def register_field(key, field):
         if GENERATE_RANDOM_ID:
             id_ = u"%d:%s" % (len(__id_store), unicode(datetime.datetime.now()))
         else:
-            id_ = unicode(hashlib.sha1("%s:%s" % (key, SECRET_SALT)).hexdigest())
+            id_ = hashlib.sha1(six.text_type("%s:%s" % (key, SECRET_SALT)).encode()).hexdigest()
```

Comment:
Can't you just use `force_unicode`/`force_text` here instead of calling `six.text_type`?